### PR TITLE
tensorflow 2.12.1 (xhochy's branch)

### PR DESCRIPTION
**Pull request by xhochy**
Continuation of https://github.com/conda-forge/tensorflow-feedstock/pull/323

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -188,35 +200,30 @@ outputs:
...
-        - tensorboard >=2.11,<2.12
-        - keras >=2.11,<2.12
+        - tensorboard >=2.12,<2.13
+        - keras >=2.12,<2.13
       run:
         - python
         - packaging
         - absl-py >=1.0.0
         - astunparse >=1.6.0
         - gast >=0.2.1,<=0.4.0
```
Comment (h-vetinari) [inline recipe/meta.yaml:210]: FYI, it turns out that fixing https://github.com/tensorflow/tensorflow/issues/56244 was just a PR away. We could consider lifting this, but it's not critical for 2.12.

**Comment by hmaarrfk** [comment]
unfortunately my mac is out of commission. sorry.

**Review by h-vetinari — APPROVED** [review]
Sounds like all the builds are done? Would be happy to get this in! :)

**Comment by hmaarrfk** [comment]
Is anybody taking charge of the linux builds? I can likely start, but I only have a single big machine. would take a 2-3 days.

**Comment by hmaarrfk** [comment]
I'm about 1/4 of the way through linux. its somewhat on autopilot so patience might be the key.

**Comment by hmaarrfk** [comment]
He did did 2.12.0 if I recall

**Comment by hmaarrfk** [comment]
Logs for linux: 
[log_files.zip](https://github.com/conda-forge/tensorflow-feedstock/files/12124991/log_files.zip)

**Pull request merged**